src/app/results/results.component.html

```diff
@@ -75,7 +75,7 @@
                 <ul class="pagination" id="pag-bar">
                     <li class="page-item"><span class="page-link" href="#" (click)="decPresentPage()">Previous</span></li>
                     <li class="page-item" *ngFor="let num of getNumber(maxPage)"><span class="page-link" *ngIf="presentPage>=4 && num<=noOfPages" [class.active_page]="getStyle(presentPage-3+num)"
-                            (click)="getPresentPage(presentPage-3+num)" href="#">{{presentPage-3+num}}</span>
+                            (click)="getPresentPage(presentPage-4+num)" href="#">{{presentPage-3+num}}</span>
```

Comment:
@nikhilrayaprolu Yes, I understood that, and I understand your intention for saying it too, but as I said if I do what you said, for pages less than 4, the pagination bar will stop working, clicking on 4 gives 3, clicking on 3 gives 2 and so on..and this is not just from 1 to 4 , every time the pagination bar updates say from 3 to 13, then the first 4 pages 3,4,5,6 will not work accordingly.
That is why I didnot change the function, and only the parameter for pages greater than 4.